assignments/life_expectancy/cleaning.py

```diff
@@ -94,15 +109,5 @@ def main() -> None: # pragma: no cover
     parser.add_argument('-o', default = '/workspaces/CF_Faast_Foundations/assignments/life_expectancy/data/pt_life_expectancy.csv', help="You need to put here the path where you want to write the output file")
```

Comment:
Is it possible to use the library path to define the arguments paths? Instead being declared completely.

Reply:
yes! I already changed and function works and pytest its ok!